clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/config/RetrofitConfig.groovy

```diff
@@ -55,6 +56,7 @@ class RetrofitConfig {
       .setConverter(new JacksonConverter())
       .setLogLevel(retrofitLogLevel)
       .setLog(new Slf4jRetrofitLogger(Front50Service))
+      .setErrorHandler(new SpinnakerRetrofitErrorHandler())
```

Comment:
I don't know if we'll ever be creating more than one of these in a single service, but this might be a good candidate for a static factory, as this is stateless so can be shared.

Reply:
:chart_with_upwards_trend: :coffee: :pouch: :scroll: :heavy_check_mark: